packages/commonwealth/client/scripts/views/components/ShareSection/getShareOptions.ts

```diff
@@ -0,0 +1,48 @@
+import mailImg from 'assets/img/share/mail.png';
+import messagesImg from 'assets/img/share/messages.png';
+import telegramImg from 'assets/img/share/telegram.png';
+import warpcastImg from 'assets/img/share/warpcast.png';
+import twitterImg from 'assets/img/share/x.png';
+
+interface ShareOption {
+  name: string;
+  icon: string;
+  onClick: () => void;
+}
+
+export const getShareOptions = (permalink: string): ShareOption[] => [
```

Comment:
we already have this implemented, please reuse https://github.com/hicommonwealth/commonwealth/blob/master/packages/commonwealth/client/scripts/views/modals/InviteLinkModal/utils.ts#L24

Reply:
My point is that we should avoid code duplication of the same functionalities, you can go ahead and modify existing share component so that it will handle mobile case as well. Either it is gonna be in this PR or upcoming, but we should have written it once and reuse everywhere.